plugins/tech-insights-backend/src/service/router.ts

```diff
@@ -191,6 +198,18 @@ export async function createRouter<
     );
   });
 
+  router.post('/facts/:namespace/:kind/:name', async (req, res) => {
+    const { namespace, kind, name } = req.params;
+    const { factRetrieverIds }: { factRetrieverIds: string[] } = req.body;
+    await factRetrieverEngine.recalculateFactsForComponent?.({
+      kind,
+      name,
+      namespace,
+      factRetrieverIds,
+    });
+    res.status(201).send();
```

Comment:
I don't know the logic deeply so I might be wrong but according to the logic this looks like a `200` 😄

Reply:
It trigger recalculation which creates new facts on the `/facts` endpoint, but does not return a response. I believe 201 is correct here